### DISCUSSION TITLE
Is there a trick to getting angular-datatables to pick up dt-options?  Using version 15.0.1.

### DISCUSSION THREAD

**Discussion by minerjed** [l-lin/angular-datatables#1749] [Q&A]
For some reason can't get dt-options to initialize.  Just keeps showing the default values.

component 
  export class WalletSubaddressComponent implements OnInit {
    dtOptions: DataTables.Settings = {};
    dtTrigger: Subject<any> = new Subject<any>();

    ngOnInit() {
      this.dtOptions = {
      "pageLength": 5
    };

html
table datatable [dtOptions]="dtOptions" [dtTrigger]="dtTrigger" 
			  class="table is-striped mx-auto is-hoverable" id="subaddid"

**Comment by shanmukhateja** [comment]
dtTrigger isn't triggered by your code to let the library its time to initialize.

Try this:

```js
ngAfterViewInit() {
    this.dtTrigger.next();
  }

````

Take a look at the Rerender Table section of the documentation:

https://l-lin.github.io/angular-datatables/#/welcome
